src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -23,6 +23,10 @@
     /** The directory where JSON files used to create data bundles are stored. */
     public static final String TEST_DATA_FOLDER = "src/test/resources/data";
 
+    /** The directory where credentials used in Gmail API are stored. */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
<!-- What's this about? Does it apply to the other fields like `TEST_DATA_FOLDER` too? -->